### PR TITLE
created portfolio for pilot data synchronization project

### PR DESCRIPTION
**Pull request by harshitha-thota**
created portfolio for pilot data synchronization project from scratch. #89

### CODE REVIEW

File: docs/portfolio.md [hunk 1]
```diff
@@ -45,13 +45,15 @@ We are especially interested in projects supporting researchers, focused on rese
   
 **[Where's Religion Mobile](project_wheres_religion_mobile/about.md)** - This Project is to create an application for ethnographers to capture the events happening around them along with some audio/visual representation of the event.  
 
+**[Pilot Data Synchronization](project_pilot_data_synchronization/about.md)** - This project aims to establish a real-time data synchronization channel between a flight simulator and the iMotions platform, enabling the accurate capture and analysis of critical flight data to enhance pilot performance evaluation.
```
Comment (kungfuchicken) [inline docs/portfolio.md:48]: This should be in alphabetical order by project title

File: docs/project_pilot_data_synchronization/about.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- **Developers:**
+ TBD
+  
+- **Start Date:** August 2024
+
+- **Technologies Used:**
+  - C++
+
+- **Type:** TBD
+- **License:** [MIT](https://opensource.org/license/mit)
```
Comment (kungfuchicken) [inline docs/project_pilot_data_synchronization/about.md:27]: Confirm that with Dr. Tamilselvan. It has implications for her. Also, we typically use LGPL as a default license. If MIT is preferred, I want to document why it's being chosen instead.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
on minor edit, one big question. you may want to change the license to TBD to get this done ASAP.

File: docs/project_pilot_data_synchronization/about.md (hunk 2 above)
Comment (harshitha-thota) [reply docs/project_pilot_data_synchronization/about.md]: I have made the necessary changes based on your feedback. Could you please review the updated code again. Thank you for your suggestions.

File: docs/portfolio.md (hunk 1 above)
Comment (harshitha-thota) [reply docs/portfolio.md]: I updated the portfolio.md file in alphabetical order by project title. Please review again.

**Review by kungfuchicken — DISMISSED** [review]
lgtm;

**Pull request merged**